doc/cephfs/troubleshooting.rst

```diff
@@ -222,6 +222,58 @@ The In-memory Log Dump can be disabled using::
 
   $ ceph config set mds mds_extraordinary_events_dump_interval 0
 
+Filesystems Become Inaccessible After an Upgrade
+================================================
+
+IF
+
+you have Ceph file systems that have data and metadata pools that were
+specified in the ``ceph fs new`` command (meaning that they were not created
```

Comment:
created by a

Reply:
Thanks for the fresh-eyed read. I've adopted this. It's better.